app/models/subject.rb

```diff
@@ -1,3 +1,5 @@
 class Subject < ApplicationRecord
+  has_many :courses, through: :course_subjects
+
```

Comment:
do we also need the association: `has_many :course_subjects` as well?

Reply:
We do. I introduced it in the next PR.